Improve plots

jonkeane opened the pull request:
Remove the lines around PUMAs on the pdf; alter proportions slightly to highlight differences more

jonkeane committed:
Improve plots

thisisnic commented on advanced_topics.qmd:
```diff
@@ -998,7 +998,7 @@ These PUMAs are designed to have approximately 100,000 people in them, and must
 As always, we start with our PUMS dataset.
 We build up a `dplyr` query to clean some data inconsistencies, such as varying capitalization, then group by language and PUMA by state and year, and calculate how many speakers of each language there are in each PUMA.
-We can see that this returns a data frame with one row per PUMA and language, with the number of speakers and the proportion of speakers compared to the overall population.
+We can see that this returns a data frame with one row per PUMA and language, with the number of speakers and the Proportion of non-English speakers compared to the overall population.
```
```suggestion
We can see that this returns a data frame with one row per PUMA and language, with the number of speakers and the proportion of non-English speakers compared to the overall population.
```

jonkeane committed:
Update advanced_topics.qmd

Co-authored-by: Nic Crane <thisisnic@gmail.com>

thisisnic reviewed (approved):
Looks great, thanks!

jonkeane commented on advanced_topics.qmd:
```diff
@@ -998,7 +998,7 @@ These PUMAs are designed to have approximately 100,000 people in them, and must
 As always, we start with our PUMS dataset.
 We build up a `dplyr` query to clean some data inconsistencies, such as varying capitalization, then group by language and PUMA by state and year, and calculate how many speakers of each language there are in each PUMA.
-We can see that this returns a data frame with one row per PUMA and language, with the number of speakers and the proportion of speakers compared to the overall population.
+We can see that this returns a data frame with one row per PUMA and language, with the number of speakers and the proportion of non-English speakers compared to the overall population.
```
```suggestion
We can see that this returns a data frame with one row per PUMA and language, with the number of speakers and the proportion of primarily non-English speakers compared to the overall population.
```

jonkeane commented on advanced_topics.qmd:
```diff
@@ -1130,12 +1131,12 @@ plot
 #| echo: false
 #| fig-height: 7
-#| fig-cap: "Proportion of speakers by language across the US by PUMA for English, Spanish, French, and Navajo"
+#| fig-cap: "Proportion of non-English speakers by language across the US by PUMA for German, French, Navajo, and Spanish"
```
```suggestion
#| fig-cap: "Proportion of primarily non-English speakers by language across the US by PUMA for German, French, Navajo, and Spanish"
```

jonkeane commented on advanced_topics.qmd:
```diff
@@ -1262,7 +1263,7 @@ plot
 #| echo: false
 #| fig-height: 5
-#| fig-cap: "Proportion of speakers by language in the Chicago by PUMA for English, Polish, and Spanish"
+#| fig-cap: "Proportion of non-English speakers by language in the Chicago by PUMA for Chinese, Polish, and Spanish"
```
```suggestion
#| fig-cap: "Proportion of primarily non-English speakers by language in the Chicago by PUMA for Chinese, Polish, and Spanish"
```

jonkeane commented:
Do those changes look alright to y'all? Or have a better way to phrase "whose primary language is not English"?

jonkeane committed:
Update advanced_topics.qmd

jonkeane committed:
Update advanced_topics.qmd

jonkeane committed:
Update advanced_topics.qmd

jonkeane committed:
Update advanced_topics.qmd

jonkeane committed:
Another reword

jonkeane commented:
Ok, I've switched to:

> ... with the number of speakers and the proportion of speakers whose primary language is not English compared to the overall population.

Which is accurate and less awkward IMHO. Still happy if someone has a better suggestion. I changed the PDF captions, but not the plot titles since we don't want those to get too long.

nealrichardson commented on advanced_topics.qmd:
```diff
@@ -1130,12 +1131,12 @@ plot
 :::
 
-We can clearly see some patterns of migration: French is spoken in parts of northern Maine and Louisiana.
+We can clearly see some patterns of migration: French is spoken in parts of northern Maine and Louisiana; German is spoken in the midwest, especially the upper midwest, and intermountain west.
```
```suggestion
We can clearly see some patterns of migration: French is spoken in parts of northern Maine and Louisiana; German is spoken in the midwest, especially the upper midwest and intermountain west.
```

nealrichardson commented on advanced_topics.qmd:
```diff
@@ -1262,7 +1263,7 @@ plot
 #| echo: false
 #| fig-height: 5
-#| fig-cap: "Proportion of speakers by language in the Chicago by PUMA for English, Polish, and Spanish"
+#| fig-cap: "Proportion of speakers whose primary language is not English by language in the Chicago by PUMA for Chinese, Polish, and Spanish"
```
```suggestion
#| fig-cap: "Proportion of speakers whose primary language is not English in the Chicago by PUMA, for Chinese, Polish, and Spanish"
```

nealrichardson commented on advanced_topics.qmd:
```diff
@@ -1344,7 +1345,7 @@ plot + facet_wrap(vars(language), ncol = 4)
 #| echo: false
 #| fig-height: 9
-#| fig-cap: "Proportion of speakers by language in the Northeast by PUMA for English, French, Portuguese, and Spanish"
+#| fig-cap: "Proportion of speakers whose primary language is not English by language in the Northeast by PUMA for Chinese, French, Portuguese, and Spanish"
```
Same idea as above. "whose primary language is not English by language in the Northeast by PUMA for" just has too many prepositions, and saying "language" twice close together just felt extra awkward.

jonkeane replied:
```suggestion
#| fig-cap: "Proportion of speakers whose primary language is not English in the Northeast by PUMA for Chinese, French, Portuguese, and Spanish"
```

Like this?

nealrichardson replied:
Yeah, better. But maybe we can drop the list of languages too? They're in the plot.

jonkeane replied:
Done (sorry for re-writting history on you there)

nealrichardson replied:
I made another edit (added a comma). WDYT?

jonkeane replied:
```suggestion
#| fig-cap: "Proportion of speakers whose primary language is not English in the Northeast, by PUMA"
```

jonkeane committed:
Update advanced_topics.qmd

Co-authored-by: Neal Richardson <neal.p.richardson@gmail.com>

jonkeane committed:
Update advanced_topics.qmd

Co-authored-by: Neal Richardson <neal.p.richardson@gmail.com>

jonkeane committed:
Update advanced_topics.qmd

jonkeane commented on advanced_topics.qmd:
```diff
@@ -1344,7 +1345,7 @@ plot + facet_wrap(vars(language), ncol = 4)
 #| echo: false
 #| fig-height: 9
-#| fig-cap: "Proportion of speakers by language in the Northeast by PUMA for English, French, Portuguese, and Spanish"
+#| fig-cap: "Proportion of speakers whose primary language is not English in the Northeast by PUMA"
```
```suggestion
#| fig-cap: "Proportion of speakers whose primary language is not English in the Northeast, by PUMA"
```

jonkeane committed:
Update advanced_topics.qmd
